photo.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html" charset="utf-8">
+	<link rel="StyleSheet" href="style.css" type="text/css">
+	<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700&display=swap&subset=cyrillic" rel="stylesheet">
+	<title>Фотогалерея</title>
+  </head>
+  <body>
+    <header class="page-header">
+	  <nav class="page-header__nav nav">
+	  	  <a class="nav__logo">
+	    <img class="nav__logo-image" src="" alt="Турагенство PINK" width="" height="">
+	  </a>
+	    <button class="main-nav__toggle" type="button">Меню</button>
+		<ul class="main-nav__list site-list">
+		  <li class="site-list__item"><a href="index.html" class="site-list__link">Главная</a></li>
+		  <li class="site-list__item"><a href="photo.html" class="site-list__link">Фотографии</a></li>
+		  <li class="site-list__item"><a href="form.html" class="site-list__link">Конкурс</a></li>
+		  <li class="site-list__item"><a href="#" class="site-list__link">HTML Академия</a></li>
+		</ul>
+	  </nav>
+	</header>
+	<main class="page-photo">
+	  <h1 class="visually-hidden">Фотогалерея</h1>
+	  <img src="" alt="" width="" height="">
+	  <section clss="page-photo__gallery gallery">
+	    <ul class="gallery__list">
+		  <li class="gallery__item--panorama post">
+		    <img class="post-img" src="" alt="" width="" height="">
+			  <blockquote class="post-item">
+		      <cite class="post__author-name">Анна</cite>
+			    <p class="post__text">Прыгать или нет? Напишите в коментарии свой совет и смотрите прямую трансляцию в перископе, задавайте свои вопросы!</p>
+		    </blockquote>
+		    <p class="post__likes"><svg></svg>Нравится</p>
```

Comment:
```suggestion
		    <div class="post__likes"><svg></svg>Нравится</div>
```

Это не абзац, там будет далеко не текстовый контент.
Обрати внимание на дизайн, что то ты забыл.